torch_xla/distributed/xla_dist.py

```diff
@@ -247,11 +267,47 @@ def _docker_run_cmd(self, cmd):
     docker_cmd.extend(cmd)
     return docker_cmd
 
+  def _tpuvm_env_vars_cmd(self, worker_idx):
+    env_vars = {
+        xenv.TPU_CHIPS_PER_HOST_BOUNDS: '2,2,1',
+        xenv.TPUVM_MODE: 1,
```

Comment:
Is this needed?

Reply:
yup, `ShouldStartLocalServerBeforeInit ` will check `TPUVM_MODE`. We only want to start the local tf server early(before XrtComputationClient creation) in the tpu_pod setup.

Reply:
Ah yes I see it thanks.